snomed/com.b2international.snowowl.snomed.datastore.tests/src/com/b2international/snowowl/snomed/core/ecl/SnomedEclEvaluationRequestHistorySupplementTest.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2022 B2i Healthcare Pte Ltd, http://b2i.sg
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.b2international.snowowl.snomed.core.ecl;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.UUID;
+
+import org.elasticsearch.core.Set;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.b2international.index.query.Expression;
+import com.b2international.snowowl.snomed.common.SnomedConstants.Concepts;
+import com.b2international.snowowl.snomed.core.domain.refset.SnomedRefSetType;
+import com.b2international.snowowl.snomed.datastore.index.entry.SnomedConceptDocument;
+import com.b2international.snowowl.snomed.datastore.index.entry.SnomedRefSetMemberIndexEntry;
+import com.b2international.snowowl.test.commons.snomed.DocumentBuilders;
+import com.b2international.snowowl.test.commons.snomed.RandomSnomedIdentiferGenerator;
+
+/**
+ * @since 8.1
+ */
+public class SnomedEclEvaluationRequestHistorySupplementTest extends BaseSnomedEclEvaluationRequestTest {
+
+	private static final String SIMPLE_REFSET_ID = RandomSnomedIdentiferGenerator.generateConceptId();
+	
+	private static final String INACTIVE_CONCEPT_1 = RandomSnomedIdentiferGenerator.generateConceptId();
+	private static final String INACTIVE_CONCEPT_2 = RandomSnomedIdentiferGenerator.generateConceptId();
+	private static final String INACTIVE_CONCEPT_3 = RandomSnomedIdentiferGenerator.generateConceptId();
+	private static final String INACTIVE_CONCEPT_4 = RandomSnomedIdentiferGenerator.generateConceptId();
+	private static final String INACTIVE_CONCEPT_5 = RandomSnomedIdentiferGenerator.generateConceptId();
```

Comment:
Please either add some comments, or rename the constants to indicate which ID corresponds to which historical association reference set. It's hard to correlate the expected values in history profile assertions.